ddp-workspace/projects/ddp-rarex/src/app/components/header/header.component.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, OnInit } from '@angular/core';
+
+import {
+  AnalyticsEventActions,
+  AnalyticsEventCategories,
+  AnalyticsEventsService,
+  SessionMementoService,
+} from 'ddp-sdk';
+
+@Component({
+  selector: 'app-header',
+  templateUrl: './header.component.html',
+  styleUrls: ['./header.component.scss']
+})
+export class HeaderComponent implements OnInit {
+  readonly naa10FoundationLink = 'https://www.naa10.org/';
+  readonly rarexLink = 'https://rare-x.org/';
```

Comment:
Similarly, does it make sense to make this configurable so it points to the dev site when we're in dev environment? Or am I misunderstanding? Is `rare-x.org` a different site and this needs to point to it?

Reply:
Right, this link points to the original Rare-X website.

Reply:
Got it, thanks for explaining!